giotto/mapper/visualization.py

```diff
@@ -54,238 +74,95 @@ def create_network_2d(graph, data, node_pos, node_color,
     fig : ploty.graph_objs.Figure
         The figure representing the nerve (topological graph).
 
-    """
-
-    # TODO: allow custom size reference
-    node_elements = graph['node_metadata']['node_elements']
-    plot_options = {
-        'edge_trace_line': dict(width=0.5, color='#888'),
-        'edge_trace_hoverinfo': None,
-        'edge_trace_mode': 'lines',
-        'node_trace_mode': 'markers',
-        'node_trace_hoverinfo': 'text',
-        'node_trace_marker_showscale': True,
-        'node_trace_marker_colorscale': 'viridis',
-        'node_trace_marker_reversescale': False,
-        'node_trace_marker_line': dict(width=.5, color='#888'),
-        'node_trace_marker_color': node_color,
-        'node_trace_marker_size': _get_node_size(node_elements),
-        'node_trace_marker_sizemode': 'area',
-        'node_trace_marker_sizeref': set_node_sizeref(node_elements),
-        'node_trace_marker_sizemin': 4,
-        'node_trace_marker_cmin': 0,
-        'node_trace_marker_cmax': 1,
-        'node_trace_marker_colorbar': dict(thickness=15,
-                                           title='',
-                                           xanchor='left',
-                                           titleside='right'),
-        'node_trace_marker_line_width': 2,
-        'node_trace_text': _get_node_text(graph),
-        'layout_showlegend': False,
-        'layout_hovermode': 'closest',
-        'layout_margin': {'b': 20, 'l': 5, 'r': 5, 't': 40},
-        'layout_xaxis': dict(showgrid=False, zeroline=False,
-                             showticklabels=False, ticks="",
-                             showline=False),
-        'layout_yaxis': dict(showgrid=False, zeroline=False,
-                             showticklabels=False, ticks="",
-                             showline=False),
-        'layout_xaxis_title': "",
-        'layout_yaxis_title': ""
-    }
-
-    if plotly_kwargs is not None:
-        plot_options.update(plotly_kwargs)
-
-    # TODO check we are not losing performance by using map + lambda
-    edge_x = list(reduce(operator.iconcat,
-                         map(lambda x: [node_pos[x[0]][0],
-                                        node_pos[x[1]][0], None],
-                             graph.get_edgelist()), []))
-
-    edge_y = list(reduce(operator.iconcat,
-                         map(lambda x: [node_pos[x[0]][1],
-                                        node_pos[x[1]][1], None],
-                             graph.get_edgelist()), []))
-
-    edge_trace = go.Scatter(
-        x=edge_x,
-        y=edge_y,
-        line=plot_options['edge_trace_line'],
-        hoverinfo=plot_options['edge_trace_hoverinfo'],
-        mode=plot_options['edge_trace_mode'])
+    Examples
```

Comment:
Create backlog entries for this and remove section for the time being

Reply:
Done